addons/im_livechat/models/res_users.py

```diff
@@ -45,7 +45,7 @@ def _inverse_livechat_lang_ids(self):
             user.res_users_settings_id.livechat_lang_ids = user.livechat_lang_ids
 
     def _compute_has_access_livechat(self):
-        for user in self:
+        for user in self.sudo():
```

Comment:
Can you use `_has_group` instead of the public method ´has_group` with sudo ?

Reply:
not really, we want to encourage people to use our helpers, not bypass it